### PR TITLE
ExpanseV2 - changed indicator to multi indicator

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/29016

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — APPROVED** [review]
I'm good with the changes. ;et's wait for a quick look of @Bargenish / @fvigo

**Review by Bargenish — APPROVED** [review]

File: Packs/ExpanseV2/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Expanse v2
+- **Breaking Change** - The **expanse-get-iprange**, **cidr** , **expanse-get-domain**, **domain**, **expanse-get-certificate**, **certificate**, **expanse-get-associated-domains**, **ip**  commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```
Comment (fvigo) [inline Packs/ExpanseV2/ReleaseNotes/1_0_1.md:4]: Not sure if we need to mention that is a breaking change: this Pack has been merged a few hours ago and is not in marketplace yet, so these commands are definitely not being used by custom scripts.

**Review by fvigo — APPROVED** [review]
Ran some tests also with the Playbook and seems to work fine, however the build fails.

File: Packs/ExpanseV2/ReleaseNotes/1_0_1.md (hunk 1 above)
Comment (bakatzir) [reply Packs/ExpanseV2/ReleaseNotes/1_0_1.md:4]: Iill add that this will affect custom scripts only.

File: Packs/ExpanseV2/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Expanse v2
+- **Breaking Change** - The **expanse-get-iprange**, **cidr** , **expanse-get-domain**, **domain**, **expanse-get-certificate**, **certificate**, **expanse-get-associated-domains**, **ip**  commands were changed to return multiple entries (entry per indicator) instead of a single entry. This change will affect custom scripts using this integration.
```
Comment (bakatzir) [inline Packs/ExpanseV2/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

**Pull request merged**